net/routing/src/opnsense/service/templates/OPNsense/Quagga/ospfd.conf

```diff
@@ -0,0 +1,49 @@
+{% macro cline(directive, modelname) -%}{% if modelname %}
+  ip ospf {{ directive }} {{ modelname }}
+{% endif %}{%- endmacro %}
+!
+! Zebra configuration saved from vty
+!   2017/03/03 20:21:04
+!
+!
+!
+!
+{% if helpers.exists('OPNsense.quagga.ospf.interfaces.interface') %}
+{%   for interface in OPNsense.quagga.ospf.interfaces.interface %}
```

Comment:
@fichtner @AdSchellevis any idea why this line is unreliable? When I have multiple interfaces configured, this will be an array of interfaces as expected but when there is a single interface, this becomes the first one (unexpected if it is an array type field in the model).